products/rhel/download.adoc

```diff
@@ -5,7 +5,7 @@
 
 To download Red Hat Enterprise Linux Developer Suite, which includes Red Hat Enterprise Linux 7 server, a collection of development tools, and link:#{site.base_url}/products/rhel/overview/[much more], you must have an account and need to accept the terms and conditions of the Red Hat Developer Program which provides $0 subscriptions for development use only. link:#{site.base_url}/faq[Read more] about the Red Hat Developers Program.
 
-==== link:https://developers.redhat.com/download-manager/link/1350474[Download Red Hat Enterprise Linux 7]
+==== link:https://developers.stage.redhat.com/download-manager/file/rhel-server-7.2-x86_64-dvd.iso[Download Red Hat Enterprise Linux 7]
```

Comment:
Wouldn't it. E better to parametrise it with site.base_url and then enable
interpolation on the adoc file? I'm concerned we forget to change it. Plus
the it will end up being wrong in staging when we want it right in
production.

On 16 Mar 2016, at 20:55, Adela Sofia Arreola <notifications@github.com>
wrote:

In products/rhel/download.adoc
<https://github.com/redhat-developer/developers.redhat.com/pull/935#discussion_r56413226>
:

> @@ -5,7 +5,7 @@
>
>  To download Red Hat Enterprise Linux Developer Suite, which includes Red Hat Enterprise Linux 7 server, a collection of development tools, and link:#{site.base_url}/products/rhel/overview/[much more], you must have an account and need to accept the terms and conditions of the Red Hat Developer Program which provides $0 subscriptions for development use only. link:#{site.base_url}/faq[Read more] about the Red Hat Developers Program.
>
> -==== link:https://developers.redhat.com/download-manager/link/1350474[Download Red Hat Enterprise Linux 7]
> +==== link:https://developers.stage.redhat.com/download-manager/file/rhel-server-7.2-x86_64-dvd.iso[Download Red Hat Enterprise Linux 7]

Yep, I think that is the plan.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub
<https://github.com/redhat-developer/developers.redhat.com/pull/935/files/fa356d88567ae17ce21d3824398536b3677e2e74#r56413226>